user-guide/Advanced_Functionality/DataMiner_Systems/BPA_tests/BPA_View_Recursion.md

```diff
@@ -4,13 +4,17 @@ uid: BPA_View_Recursion
 
 # View Recursion
 
+<!-- # View Recursive Loops
+
+Prior to DataMiner 10.4.12/10.5.0, this BPA test is called "View Recursion".-->
+
 This BPA will check the *Views.xml* file to see if any loops are present.
 
-This BPA test is available by default from DataMiner 10.1.0 [CU4]/10.1.7 onwards.
+This BPA test is available by default from DataMiner 10.1.0 [CU4]/10.1.7 onwards. However, as of DataMiner 10.4.12/10.5.0<!--RN 40751-->, it is no longer included as a standard BPA test in the *System Center* > *Agents* > *bpa* overview and cannot be executed.
```

Comment:
As mentioned in my first comment above, we should either make clear where the test is available so it can still be executed from there, or we should simply mention that it is no longer available from this version onwards.

Reply:
```suggestion
This BPA test is available by default from DataMiner 10.1.0 [CU4]/10.1.7 up to DataMiner 10.4.12/10.5.0<!--RN 40751-->.
```